doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -910,6 +910,25 @@ Memfault integration
 * Removed a metric for the tracking Bluetooth TX thread unused stack ``ncs_bt_tx_unused_stack``.
   The thread in question was removed in Zephyr v3.7.0.
 
+* Simplified the options for ``CONFIG_MEMFAULT_NCS_DEVICE_ID_*``, which sets the Memfault Device Serial:
+
+  * :kconfig:option:`CONFIG_MEMFAULT_NCS_DEVICE_ID_HW_ID` (new default) - use `hw_id` provided device ID, which is also what nRF Cloud uses for device identity
+  * :kconfig:option:`CONFIG_MEMFAULT_NCS_DEVICE_ID_STATIC` used to set a custom build-time defined static device id, primarily only useful for testing
+  * :kconfig:option:`CONFIG_MEMFAULT_NCS_DEVICE_ID_RUNTIME` use a runtime-applied device id, commonly used when device serial is written into settings at manufacturing time, for example
+
+  Additionally, when using ``hw_id`` as the Memfault Device Serial source, the default ``hw_id`` source is modified depending on system configuration. In order:
+
+  * If `MEMFAULT_NCS_DEVICE_ID_IMEI || NRF_MODEM_LIB` is selected, use IMEI. Note that :kconfig:option:`CONFIG_MEMFAULT_NCS_DEVICE_ID_IMEI` is kept for backwards compatibility, but is marked deprecated
+  * Else if `BOARD_NRF7002DK_NRF5340_CPUAPP` is selected, use network MAC address (special case for backwards compatibility)
+  * Else if `BT` is selected, use Bluetooth MAC address
+  * Otherwise, the default `HW_ID_LIBRARY_SOURCE_DEVICE_ID` (chip UID) will be used
```

Comment:
```suggestion
  * If ``BOARD_NRF7002DK_NRF5340_CPUAPP`` is selected, use the network MAC address (special case for backwards compatibility).
  * If ``BT`` is selected, use the Bluetooth MAC address.
  * Otherwise, the default ``HW_ID_LIBRARY_SOURCE_DEVICE_ID`` (chip UID) will be used.
```